packages/button/src/component.tsx

```diff
@@ -28,25 +28,73 @@ export const Button = forwardRef<
+    // Disabled
+    buttonIsDisabled:
+      'font-bold max-w-max justify-center transition-colors ease-in-out i-bg-$color-button-disabled-background! i-text-$color-button-disabled-text cursor-default pointer-events-none', // .button:disabled, .button--is-disabled
+    primaryDefault: 'border-0 rounded-8 font-bold max-w-max focusable justify-center transition-colors ease-in-out i-text-$color-button-primary-text hover:i-text-$color-button-primary-text i-bg-$color-button-primary-background hover:i-bg-$color-button-primary-background-hover! active:i-bg-$color-button-primary-background-active',
+    secondaryDefault: 'border-2 font-bold rounded-8 max-w-max focusable justify-center transition-colors ease-in-out i-text-$color-button-secondary-text hover:i-text-$color-button-secondary-text i-border-$color-button-secondary-border i-bg-$color-button-secondary-background hover:i-bg-$color-button-secondary-background-hover hover:i-border-$color-button-secondary-border-hover active:i-bg-$color-button-secondary-background-active'
+  }
+    
+  const ccButton = {
+    // Buttontypes
+    buttonSecondary:
+    'py-10 px-14 text-m leading-[24] ' + buttonVariants.secondaryDefault,
+    buttonSecondarySmall:
+    'py-6 px-16 text-xs ' + buttonVariants.secondaryDefault,
+    buttonSecondaryQuiet:
+    'py-10 px-14 text-m leading-[24] ' + buttonVariants.buttonFlat ,
+    buttonSecondarySmallQuiet:
+    'py-6 px-16 text-xs ' + buttonVariants.buttonFlat,
+    buttonSecondaryLoading:
+    'py-10 px-14 text-m leading-[24] ' + buttonVariants.secondaryDefault + buttonVariants.buttonInProgress,
+    buttonSecondarySmallLoading:
+    'py-6 px-16 text-xs ' + buttonVariants.secondaryDefault + buttonVariants.buttonInProgress,
+    buttonSecondarySmallQuietLoading:
+    'py-6 px-16 text-xs ' + buttonVariants.buttonFlat + buttonVariants.buttonInProgress,
+    buttonSecondaryQuietLoading:
+    'py-10 px-14 text-m leading-[24] ' + buttonVariants.buttonFlat + buttonVariants.buttonInProgress,
+    buttonPrimary:
+      'py-12 px-16 text-m leading-[24] ' + buttonVariants.primaryDefault,
+    buttonPrimaryQuiet:
+      'py-12 px-16 text-m leading-[24] ' + buttonVariants.buttonFlat,
+    buttonPrimarySmall:
+      'py-8 px-16 text-xs ' + buttonVariants.primaryDefault,
+    buttonPrimarySmallQuiet:
+      'py-8 px-16 text-xs ' + buttonVariants.buttonFlat,
+    buttonPrimaryLoading:
+      'py-12 px-16 text-m leading-[24] ' + buttonVariants.primaryDefault + buttonVariants.buttonInProgress,
+    buttonPrimarySmallLoading:
+      'py-8 px-16 text-xs  ' + buttonVariants.primaryDefault + buttonVariants.buttonInProgress,
+    buttonPrimaryQuietLoading:
+      'py-12 px-16 text-m leading-[24]  ' + buttonVariants.buttonFlat + buttonVariants.buttonInProgress,
```

Comment:
This can also be re-used
```suggestion
     buttonVariants.primarySizing + buttonVariants.buttonFlat + buttonVariants.buttonInProgress,
```